modules/softwarelandscape/slides.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Software Landscape
+type: slides
+order: 1
+---
+
+<!-- .slide: data-state="title" -->
+
+# Software in the Scientific Landscape
+
+===
+
+<!-- .slide: data-state="standard center" -->
+## Some general observations:
```

Comment:
I like the train of thought here but it is quite a lot to have on one slide

Reply:
This was addressed in https://github.com/esciencecenter-digital-skills/research-software-support/pull/121/commits/7aefb650908186ae75b13d7a1205cf9f9779adc2